runtime/ejs-webgl.m

```diff
@@ -1320,8 +1349,20 @@ any getFramebufferAttachmentParameter(GLenum target, GLenum attachment,
  GLboolean isRenderbuffer(WebGLRenderbuffer renderbuffer);
  GLboolean isShader(WebGLShader shader);
  GLboolean isTexture(WebGLTexture texture);
- void lineWidth(GLfloat width);
  */
+
+//  void lineWidth(GLfloat width);
+JSMETHOD(lineWidth) {
+    if (argc != 1)
+        THROW_ARG_COUNT_EXCEPTION(1);
+
+    GLfloat width = (GLfloat)EJSVAL_TO_NUMBER (args[0]);
```

Comment:
doesn't have to be part of this PR, but I think we should be doing ToDouble/ToInt32/etc instead of just EJSVAL_TO_NUMBER.  The latter will give really bizarre results if anything but a number is passed.